src/lib/api/app.js

```diff
@@ -16,7 +16,8 @@ export default async function( app: string | number, fields: ?any ): Promise<any
 	}
 
 	const api = await API();
-	if ( isNaN( parseInt( app ) ) ) {
+	if ( isNaN( app ) ) {
+		app = parseInt( app, 10 );
```

Comment:
I don't think we want the `parseInt` here since it's the app `name` (a string). We need it for the `id` condition below.

Reply:
Oh, good catch. I guess I only tested with an ID 🙃